### PR TITLE
fix: Wallet History Page performance

### PR DESCRIPTION
**Pull request by staylorwr**
Related to radixdlt/radixdlt-javascript#209.

This PR updates the wallet for the changes included in 209 which drops the
AbstractAddressT types from the ledger's transaction response handlers and
instead returns the string representation of those addresses.

### CODE REVIEW

File: src/composables/useHistory.ts [hunk 1]
```diff
@@ -69,6 +69,7 @@ export default function useHistory (radix: ReturnType<typeof Radix.create>): use
   }
 
   const pushMsg = (tx: ExecutedTransaction, msg: string) => {
+    console.log(tx, msg, 'hey i got this stuff')
```
Comment (P-v-R) [inline src/composables/useHistory.ts:72]: 🚨 👮🏻‍♂️ Console.log Police 👮🏻‍♂️🚨

File: src/views/Wallet/WalletHistory.vue [hunk 2]
```diff
@@ -160,8 +149,10 @@ const WalletHistory = defineComponent({
     })
 
     const activateThenDecrypt = async (data: ExecutedTransaction) => {
+      console.log('oh hi im decrypting')
```
Comment (P-v-R) [inline src/views/Wallet/WalletHistory.vue:152]: 👮🏻‍♂️ 👮🏻‍♂️ 👮🏻‍♂️ 👮🏻‍♂️

File: src/composables/useWallet.ts [hunk 3]
```diff
@@ -74,6 +75,7 @@ export interface PendingTransaction extends TransactionStateSuccess {
 export type AmountFormats = 'us' | 'europe'
 
 const radix = Radix.create()
+radix.logLevel(LogLevel.WARN)
```
Comment (P-v-R) [inline src/composables/useWallet.ts:78]: what is the purpose of this?

File: src/helpers/formatter.ts [hunk 4]
```diff
@@ -7,7 +7,11 @@ export const formatWalletAddressForDisplay = function (address: AccountAddressT,
 
 export const formatValidatorAddressForDisplay = function (address: ValidatorAddressT): string {
   const s = address.toString()
-  return s.substring(0, 2) + '...' + s.substring(s.length - 9)
+  return formatValidatorAddressString(s)
+}
+
+export const formatValidatorAddressString = function (address: string): string {
```
Comment (artsycoder533) [inline src/helpers/formatter.ts:13]: same here, creating this function to abstract out some of the functionality other places.

**Review by artsycoder533 — APPROVED** [review]
^^ What Perry said regarding the stray console logs.  Other than that, It looks like you made the code cleaner too!  I feel like the history page loads faster as well, if I remember correctly

**Pull request merged**